drivers/platform/aducm3029/aducm3029_timer.h

```diff
@@ -0,0 +1,105 @@
+/***************************************************************************//**
+ *   @file   aducm3029_timer.h
+ *   @brief  Header file of TIMER driver for ADuCM302x
+ *   @author Mihail Chindris (mihail.chindris@analog.com)
```

Comment:
is this the correct author? was this file only moved?

Reply:
this file was renamed and I added some new changes.